src/styles/main.scss

```
@@ -1,3 +1,65 @@
+$day-size: 100px;
+$col-numbers: 7;
+$gap: 1px;
+
 body {
   margin: 0;
 }
+
+.calendar {
+  margin: 30px;
+  display: flex;
+  gap: 1px;
+  flex-wrap: wrap;
+  width: calc(#{$day-size * $col-numbers + 10px});
+
+  $first-days-shifts: mon 0,
+    tue calc(#{$day-size + $gap}),
+    wed calc(2 * #{$day-size + $gap} ),
```

```suggestion
    tue 1,
    wed 2,
```
and calculate it only one time in the each (line 26)